composer.json

```diff
@@ -52,6 +52,7 @@
 		"helmich/typo3-typoscript-lint": "^3.1.1",
 		"icanhazstring/composer-unused": "^0.8.11",
 		"php-coveralls/php-coveralls": "^2.7.0",
+		"phpmd/phpmd": "^2.15.0",
```

Comment:
composer normalize has something to complain about.  So let's normalize the `composer.json`.